editor/editor_properties.cpp

```diff
@@ -128,6 +128,17 @@ void EditorPropertyText::set_placeholder(const String &p_string) {
 	text->set_placeholder(p_string);
 }
 
+void EditorPropertyText::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_THEME_CHANGED: {
+			if (prefix) {
+				Color font_color = get_theme_color(SNAME("disabled_font_color"), EditorStringName(Editor));
+				prefix->set_modulate(font_color);
+			}
+		}
```

Comment:
```suggestion
		} break;
```

Reply:
Amending suggested change into existing commit to avoid commits with a broken state. Thanks!